code/modules/admin/verbs/adminsay.dm

```diff
@@ -42,5 +42,5 @@
 		sender_name = "<span class='admin'>[sender_name]</span>"
 	for(var/client/C in admins)
 		C << "<span class='mod_channel'>" + create_text_tag("mod", "MOD:", C) + " <span class='name'>[sender_name]</span>([admin_jump_link(mob, C.holder)]): <span class='message'>[msg]</span></span>"
-		C << "<span class='dev_channel'>" + create_text_tag("dev", "DEV:", C) + " <span class='name'>[sender_name]</span>(<A HREF='?src=\ref[C.holder];adminplayerobservejump=\ref[mob]'>JMP</A>): <span class='message'>[msg]</span></span>"
+		C << "<span class='dev_channel'>" + create_text_tag("dev", "DEV:", C) + " <span class='name'>[sender_name]</span>([admin_jump_link(mob, C.holder)]): <span class='message'>[msg]</span></span>"

```

Comment:
That's not what I meant. This section shouldn't be here. Look at how mod and adminsay does it.